Brokerages/GDAX/GDAXBrokerage.Messaging.cs

```diff
@@ -321,66 +323,38 @@ private void OnL2Update(string data)
             }
         }
 
-        private void OrderMatch(string data)
+        private void OnMatch(string data)
         {
             // deserialize the current match (trade) message
             var message = JsonConvert.DeserializeObject<Messages.Matched>(data, JsonSettings);
 
-            if (_isDataQueueHandler)
+            if (string.IsNullOrEmpty(message.UserId))
             {
-                EmitTradeTick(message);
+                // message received from the "matches" channel
```

Comment:
> message received from the "matches" channel

But we are not subscribing to `matches` anymore?

Reply:
Correct, `GDAXBrokerage` does not subscribe to `matches` anymore, but `ChannelNames` is `virtual` and is overridden in `GDAXDataQueueHandler` which has `matches` in the list.